.circleci/config.yml

```diff
@@ -547,6 +547,26 @@ jobs:
       - run:
           name: Install Desktop Dependencies
           command: yarn run build-desktop:install-app-deps
+      - run:
+          name: Build Source
+          command: |
+            make -f desktop/Makefile build-main
+      # - when:
+          # condition: << pipeline.git.tag >>
+          # steps:
+      - run:
+          name: Build Appx
+          command: |
+            # Note: Windows Store build should be unsigned
+            make -f desktop/Makefile package ELECTRON_BUILDER_ARGS='--config="electron-builder-appx.json"'
+      - run:
+          name: Clean Up Appx
```

Comment:
@sandymcfadden This change seems to work to build the unsigned .appx. However there are a bunch of additional artifacts that are generated that I can't seem to delete. These cleanup commands work in a local Powershell instance and when I SSH into the Circle machine, however for some reason the additional artifacts aren't being cleaned up when the CI script actually runs.